app/packages/embeddings/src/tracesToData.tsx

```diff
@@ -70,3 +73,48 @@ export function tracesToData(
       };
     });
 }
+
+const getLabelColor = (key: string, setting: CustomizeColor) => {
+  if (setting && setting.labelColors) {
+    const color = setting.labelColors.find((x) => x.name === key)?.color;
+    return getConvertedColor(color);
+  }
+};
+
+const getConvertedColor = (color: string | number[]) => {
```

Comment:
I think the return value here should be explicitly typed, or the conditionals cleaned up. It currently says it strictly returns `Color`